File: testdata/lang/packages/dst/geom/geom.c
#include "geom.h"

// -- Forward declarations (functions and methods) --
static double rectArea(double width, double height);

// -- Implementation --
const double geom_Pi = 3.14159;

static double rectArea(double width, double height) {
    return width * height;
}

double geom_RectArea(double width, double height) {
    return rectArea(width, height);
}

File: testdata/packages/src/geom/geom.go
package geom

const Pi = 3.14159

type Rect struct {
	W, H float64
}

func (r Rect) Area() float64 {
	return rectArea(r.W, r.H)
}

func rectArea(width, height float64) float64 {
	return width * height
}

func RectArea(width, height float64) float64 {
	return rectArea(width, height)
}
